src/middlewared/middlewared/plugins/pool.py

```diff
@@ -1473,14 +1433,7 @@ async def import_pool(self, job, data):
                             await delegate.toggle(attachments, True)
             await self.middleware.call('keyvalue.delete', key)
 
-        await self.middleware.call('service.reload', 'disk')
-        asyncio.ensure_future(self.restart_services())
```

Comment:
Btw `collectd` is restarted when this is called, that won't happen now - is that okay ?